<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>937</width>
    <height>653</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Usbip Gui</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../main_icon.ico</normaloff>../main_icon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="styleSheet">
       <string/>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="handleWidth">
       <number>4</number>
      </property>
      <widget class="QFrame" name="frame_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string/>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="leftMargin">
         <number>3</number>
        </property>
        <property name="topMargin">
         <number>3</number>
        </property>
        <property name="rightMargin">
         <number>3</number>
        </property>
        <property name="bottomMargin">
         <number>3</number>
        </property>
        <item row="0" column="0">
         <widget class="QTreeWidget" name="usb_tree_widget">
          <property name="font">
           <font>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="lineWidth">
           <number>1</number>
          </property>
          <property name="showDropIndicator" stdset="0">
           <bool>true</bool>
          </property>
          <property name="iconSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="indentation">
           <number>40</number>
          </property>
          <property name="headerHidden">
           <bool>true</bool>
          </property>
          <attribute name="headerVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>11</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QFrame" name="frame">
       <property name="styleSheet">
        <string/>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>3</number>
        </property>
        <property name="topMargin">
         <number>3</number>
        </property>
        <property name="rightMargin">
         <number>3</number>
        </property>
        <property name="bottomMargin">
         <number>3</number>
        </property>
        <item row="0" column="0">
         <widget class="QTextEdit" name="log_text_edit">
          <property name="styleSheet">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>937</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="about_action"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Действия</string>
    </property>
    <addaction name="add_server_action"/>
    <addaction name="remove_server_action"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Настройки</string>
    </property>
    <addaction name="filters_action"/>
   </widget>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu"/>
  </widget>
  <action name="open_about_action">
   <property name="text">
    <string>О программе...</string>
   </property>
  </action>
  <action name="about_action">
   <property name="text">
    <string>О программе...</string>
   </property>
  </action>
  <action name="add_server_action">
   <property name="icon">
    <iconset resource="../qt_utils/resources/icons.qrc">
     <normaloff>:/icons/icons/add_server.png</normaloff>:/icons/icons/add_server.png</iconset>
   </property>
   <property name="text">
    <string>Добавить сервер</string>
   </property>
  </action>
  <action name="remove_server_action">
   <property name="icon">
    <iconset resource="../qt_utils/resources/icons.qrc">
     <normaloff>:/icons/icons/remove_server.png</normaloff>:/icons/icons/remove_server.png</iconset>
   </property>
   <property name="text">
    <string>Удалить сервер</string>
   </property>
  </action>
  <action name="filters_action">
   <property name="icon">
    <iconset resource="../qt_utils/resources/icons.qrc">
     <normaloff>:/icons/icons/filters.png</normaloff>:/icons/icons/filters.png</iconset>
   </property>
   <property name="text">
    <string>Общие фильтры</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../qt_utils/resources/icons.qrc"/>
 </resources>
 <connections/>
</ui>
